backend/src/controllers/task.js

```diff
@@ -25,16 +25,8 @@ router.get('/tasks', async (req, res) => {
 
   // DELETE task
 router.delete('/tasks/:id', async (req, res) => {
-  console.log(req.params.id)
```

Comment:
Remember to remove all `console.log()`s that are only there for debugging purposes before commiting :)